gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageMockitoTest.java

```diff
@@ -77,7 +77,13 @@ public void setUp() {
     MockitoAnnotations.initMocks(this);
 
     MockHttpTransport transport = mockTransport();
-    gcs = mockedGcs(transport);
+    final GoogleCloudStorageOptions GCS_OPTIONS =
```

Comment:
We usually do not use final inside the methods, and all vars should be camelCase except class-level constants.